src/renderer/views/wallet/PoolShareView.tsx

```diff
@@ -106,7 +111,8 @@ export const PoolShareView: React.FC = (): JSX.Element => {
   const clickRefreshHandler = useCallback(() => {
     reloadPools()
     reloadNetworkInfo()
-  }, [reloadNetworkInfo, reloadPools])
+    reloadCombineSharesByAddresses()
```

Comment:
missed for error-view 
![image](https://user-images.githubusercontent.com/67793194/112624029-1f596180-8e3e-11eb-8ae9-2cafc7f885ec.png)